crates/forge-runner/src/running.rs

```diff
@@ -361,11 +363,18 @@ fn get_latest_block_number(url: &Url) -> Result<BlockId> {
 }
 
 fn get_all_execution_resources(runtime: &ForgeRuntime) -> ExecutionResources {
-    let test_used_resources = &runtime.0.extended_runtime.child.child.resources;
-    let cheatnet_used_resources = &runtime
+    let test_used_resources = &runtime
         .0
         .extended_runtime
+        .0
+        .get_extended_runtime()
         .child
+        .resources;
+    let cheatnet_used_resources = &runtime
+        .0
+        .extended_runtime
+        .0
```

Comment:
With the nesting this can quickly get tedious - can we implement `Deref` for the runtime tuples?

Reply:
Removed in the next PR, resolving